locales/sv.js

```diff
@@ -1,316 +1,632 @@
+promo_code_is_expired:
+"Kampanjkoden har löpt ut.",
+promo_code_code:
+"Har du en kampanjkod?",
+promo_code_rate_on_order:
+"Rabatt på din order",
+promo_code_alternate_price:
+"Specialpris på produkter",
+total:
+"Totalt",
+total_paid:
+"Total paid",
+province_state:
+"Län",
+billing_address:
+"Faktureringsadress",
+shipping_address:
+"Leveransadress",
+payment_informations:
+"Betalningsinformation",
+payment_informations_bill_me_later:
+"Jag vill betala mot faktura",
+payment_informations_no_payment_required:
+"Ingen betalning krävs",
+payment_informations_paypalexpress:
+"Betala med Paypal",
+credit_card_type_mastercard:
+"Mastercard",
+credit_card_type_visa:
+"Visa",
+credit_card_type_amex:
+"American Express",
+months_january:
+"Januari",
+months_february:
+"Februari",
+months_march:
+"Mars",
+months_april:
+"April",
+months_may:
+"Maj",
+months_june:
+"Juni",
+months_july:
+"Juli",
+months_august:
+"Augusti",
+months_september:
+"September",
+months_october:
+"Oktober",
+months_november:
+"November",
+months_december:
+"December",
+cart_items_table_item:
+"Artikel",
+cart_items_table_description:
+"Beskrivning",
+cart_items_table_quantity:
+"Antal",
+cart_items_table_unit_price:
+"Pris",
+cart_items_table_total_price:
+"Totalpris",
+cart_empty_text:
+"Kundvagnen är tom, välj en produkt innan du går till kassan",
+new_account_form_create_new_account:
+"Skapa en inloggning",
+new_account_form_create_new_account_action:
+"Skapa en inloggning",
+login_form_having_an_account:
+"Logga in",
+login_form_login_action:
+"Logga in",
+login_title:
+"Inloggning",
+login_form_forgot_password_action:
+"Jag har glömt mitt lösenord",
+forgot_password_forgot_your_password:
+"Har du glömt ditt lösenord?",
+forgot_password_please_enter_email:
+"Ange din e-postadress så skickar vi ett e-postmeddelande med en unik länk för att återställa ditt lösenord.",
+forgot_password_success_email_sent:
+"E-post skickat",
+forgot_password_email_sent_message:
+"Ett e-postmeddelande har skickats till dig med instruktioner om hur du återställer ditt lösenord. Vänligen kolla din e-post och följ anvisningarna.",
+login_checkout_as_guest:
+"Till kassan som en gäst",
+login_checkout_as_guest_notice:
+"Vid slutet av kassan, kommer du att erbjudas möjligheten att skapa ett konto med hjälp av den information du angett tidigare.",
+shipping_address_same_as_billing:
+"Samma som fakturaadress",
+shipping_method_method_name:
+"Fraktalternativ",
+shipping_method_shipping_price:
+"Fraktkostnad",
+shipping_method_failure_message:
+"Vi har tyvärr inte kunnat hitta något möjligt fraktalternativ. Vänligen kolla att din leveransadress är korrekt och försök igen.",
+shipping_method_failure_click_here_to_edit:
+"Klicka här för att ändra din leveransadress",
+payment_method_card_holder:
+"Namn på kortet",
+payment_method_card_type:
+"Korttyp",
+payment_method_card_number:
+"Kortnummer",
+payment_method_card_cvc:
+"CVC",
+payment_method_card_exp_month:
+"Giltigt till månad",

```

Comment:
Why remove year?

Reply:
payment_method_card_exp_year exist in file

Reply:
I missed that. All good, thanks